packages/cpp/README.md

```diff
@@ -102,6 +102,28 @@ There are two ways to configure clang-tidy's checks: through a Theia preference
     - for the .clang-tidy file: Checks: "-*,readability-*"
         - Meaning: disable all list-checks and enable all readability-* checks
 
+### Using clang-tidy as a task
```

Comment:
Do you think `using` or `running` is more appropriate?
I think the wording here should state `Running clang-tidy as a task`.

Reply:
Changed